plugins/foreman_remote_execution/0.1/index.md

```diff
@@ -0,0 +1,394 @@
+
+Alternatively, the Foreman proxy publishes the public key it over the
+API, so that one can just download it into authorized keys:
+
+    curl https://myproxy.example.com:8443/ssh/pubkey >> ~/.ssh/authorized_keys
+
+For the execution to work, the client needs to have **openssh-server**
+installed and configured.  Also, **openssh-clients** need to be present, in
+order to **scp** to work (which we use for getting
+the scripts to the remote host)
+
+
+# 3. Usage
+
+After installation you can start preparing your commands and run them
+on your hosts. Every command must be defined as a Job Template. Once
+defined it can be executed multiple times. Let's start with the
+definition first.
+
+## 3.1 Job Templates
+
+If you're familiar with Foreman provisioning templates, you should
+find writing job templates familiar. First navigate to Host > Job
+Templates (new menu entry). Then hit the green button with label "New
+Job Template".
+
+Note: If you don't see menu entry, the Foreman plugin is not installed
+correctly.
+
+Now you should see form similar to Provisioning template one. In first
+tab you have to fill in **Name**. You can use input macros in ERB tags
+which stands for variables which you define in second tab called Job.
+
+![Job Template form](/plugins/foreman_remote_execution/{{page.version}}/job_template_form.png)
+
+Before we get to inputs, lets look at other fields in Job tab. First
+one is autocomplete field **Job name**. This is generic name of the
+Job, which might be implemented by more Job Templates: for example one
+template can use use `yum` to install packages while second one uses
+`dnf`. **Job name** should be "install package" for both, while
+**Name** attributes would be something like "install package using
+yum" and "install package using dnf". This will become more handy when
+there are more than one **Provider type**. **Provider type** is
+another attribute which you don't have to change unless you install
+additional connection provider plugins. If you're unsure about the
+difference between **Name** and **Job name**, you can keep both the
+same, later you'll be using **Job name** to select which job you want
+to execute.
+
+On the very same tab you can configure inputs. These template inputs
+defines values that can be accessed in the template code by input
+macros based on it's name. To add new input, click green button "+ Add
+Input" which add more fields to the form. To remove the input, hit the
+cross in top right corner of the input form.
+
+We support four types of inputs. Each must have it's **name** which is
+used in macro. Let's say my input has name "motd", I can then use
+following ERB tag in my template code
+
+    echo "<%= input('motd') -%>" > /etc/motd
+
+When template is rendered for particular host, the `input('motd')` is
+evaluated like this:
+
+1. find and input with name motd
+2. determine its type
+3. based on type, fetch the value
```

Comment:
This isn't really mine, the reason there's two commits is so you view the diff of what this PR changes from 0.0 manual.  I'll fix it I guess...